library/multipicker/src/main/java/im/vector/lib/multipicker/utils/ImageUtils.kt

```diff
@@ -23,36 +23,42 @@ import android.graphics.ImageDecoder
 import android.net.Uri
 import android.os.Build
 import androidx.exifinterface.media.ExifInterface
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
 import timber.log.Timber
 
 object ImageUtils {
 
-    fun getBitmap(context: Context, uri: Uri): Bitmap? {
-        return try {
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
-                ImageDecoder.decodeBitmap(ImageDecoder.createSource(context.contentResolver, uri))
-            } else {
-                context.contentResolver.openInputStream(uri)?.use { inputStream ->
-                    BitmapFactory.decodeStream(inputStream)
+    suspend fun getBitmap(context: Context, uri: Uri): Bitmap? {
+        return withContext(Dispatchers.IO) {
```

Comment:
the diffs in this PR are a bit painful, this block has been wrapped with `withContext { }`